examples/getContent.js

```diff
@@ -11,5 +9,6 @@ github.repos.getContent({
   repo: 'node-github',
   path: ''
 }, function (err, res) {
-  console.log(err, res)
+  //console.log(err, res)
+  console.log("Error thrown in getContent.js");
```

Comment:
you can remove the callback altogether :) See https://github.com/octokit/node-github/blob/624/run-examples-as-part-of-tests/examples/addCollaborator.js as an example